### PR TITLE
FPO-255 Add GovNotify configuration

### PR DESCRIPTION
**Pull request by mkayad**
### Jira link
[FPO-255](https://transformuk.atlassian.net/browse/FPO-255)
### What?

I have added/removed/altered:

- [x] Added email configuration
- [x] Removed
- [x] Altered

### Why?

I am doing this because:

- we need to inform the user the progress of their application

### AC

- Tested
- Shared with stakeholders
- Pair reviewed

### CODE REVIEW

File: src/controllers/verificationController.ts [hunk 1]
```diff
@@ -93,9 +95,18 @@ export const applicationComplete = async (req: Request, res: Response): Promise<
   const body = req.body
   const session = req.session ?? {}
   const result = validationResult(req)
+  const templateId: string = process.env.REGISTRATION_TEMPLATE_ID ?? ''
```
Comment (willfish) [inline src/controllers/verificationController.ts:98]:
```suggestion
  const registrationTemplateId: string = process.env.REGISTRATION_TEMPLATE_ID ?? ''
```

File: globals.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+declare module 'notifications-node-client'
```
Comment (willfish) [inline globals.d.ts:1]: Is this because this type is missing from the upstream package?

File: src/services/emailService.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-redeclare */
+
+import { logger } from '../config/logging'
+import { NotifyClient } from 'notifications-node-client'
+
+const govUKNotifyAPIKey = process.env.GOVUK_NOTIFY_API_KEY ?? ''
+
+type NotifyClient = typeof NotifyClient
```
Comment (willfish) [inline src/services/emailService.ts:8]: I don't understand why we need this chipolata

File: globals.d.ts (hunk 2 above)
Comment (mkayad) [reply globals.d.ts:1]: yes, that is correct

File: src/services/emailService.ts (hunk 3 above)
Comment (mkayad) [reply src/services/emailService.ts:8]: I got a compilation error in the constructor of NotifyClient without this alias

**Review by willfish — APPROVED** [review]

**Pull request merged**